components/leaflet-maps/NormalMap.vue

```diff
@@ -11,29 +11,33 @@
 
     <l-marker v-for="(marker, index) in mapMarkers" :key="index" :lat-lng="marker.longLat">
       <l-popup class="adapted-popup">
-
         <h2>{{marker.location}}</h2><br>
         <div :class="marker.witches.length > 1 ? 'witch-scroller' : 'no-witch-scroller'">
           <div v-for="(witch, index) in marker.witches" :key="index">
 
-            <strong>{{ witch.name }}</strong><br>
+            <div class="font-semibold text-base">{{ witch.name }}</div><br>
             <div>
-              Investigation Date: {{ witch.investigationDates[1] }}<br>
+              <b>Investigation Date:</b> {{ witch.investigationDates[1] }}<br>
             </div>
-            Gender: {{ witch.sex }}<br>
-            Occupation: {{ witch.occupation }}<br>
-            Social Class: {{ witch.socialClass }}<br>
+            
+            <div v-for="standardAttribute in getStandardsWithValue(witch)">
+                <b>{{standardLabels[standardAttribute]}}:</b>
+                {{ witch[standardAttribute] }}
+                <br>
+              </div>
 
             <div v-for="locationOption in getLocationsWithValue(witch)">
-              {{locationsLabels[locationOption]}}:
-              <a @click="flyTo(witch[locationOption].coordinates)" :style="{ cursor: 'pointer'}">{{
-                witch[locationOption].location }}
-              </a>
+              <b>{{locationsLabels[locationOption]}}:</b>
+              <template v-for="(subLocation, index) in witch[locationOption].locations">
+                <a @click="flyTo(witch[locationOption].coordinates[index])" :style="{ cursor: 'pointer'}">{{subLocation }}
```

Comment:
Please add space so {{subLocation }} is {{ subLocation }}. Just for consistency.